install/sample/base/kustomization.yaml

```diff
@@ -1,8 +1,6 @@
 apiversion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: sample
```

Comment:
not using namespace anymore?

Reply:
adding it back, I'm not sure how this got removed